packages/app/obojobo-document-engine/src/scripts/oboeditor/components/navigation/more-info-box.js

```diff
@@ -48,13 +49,26 @@ class MoreInfoBox extends React.Component {
 		this.close()
 	}
 
+	componentDidUpdate() {
+		// This autofocuses the id input box when the node is opened
+		// for easy use by keyboard users
+		if (this.state.isOpen || this.props.open) {
+			document.addEventListener('mousedown', this.handleClick, false)
+			this.idInput.focus()
```

Comment:
I think we should also `setTimeout(() => this.idInput.select(), 1)`

Reply:
I think we need to only do this when prevState.isOpen !== this.state.isOpen. When trying to (for example) change a page title the focus keeps returning to the id input